lib/actions/ssh.ts

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2016-2017 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CommandDefinition } from 'capitano';
+import * as commandOptions from './command-options';
+
+export = <CommandDefinition<
+	{
+		uuid?: string;
+	},
+	{
+		port?: number;
+		host?: boolean;
+		verbose?: boolean;
+		noproxy?: boolean;
+	}
+>>{
+	signature: 'ssh [uuid]',
+	description: '(beta) get a shell into the running app container of a device',
+	help: `\
+Warning: 'resin ssh' requires an openssh-compatible client to be correctly
+installed in your shell environment. For more information (including Windows
+support) please check the README here: https://github.com/resin-io/resin-cli
+
+Use this command to get a shell into the running application container of
+your device.
+
+Examples:
+
+	$ resin ssh MyApp
+	$ resin ssh 7cf02a6
+	$ resin ssh 7cf02a6 --port 8080
+	$ resin ssh 7cf02a6 -v
+	$ resin ssh 7cf02a6 -s\
+`,
+	permission: 'user',
+	primary: true,
+	options: [
+		{
+			signature: 'port',
+			parameter: 'port',
+			description: 'ssh gateway port',
+			alias: 'p',
+		},
+		{
+			signature: 'verbose',
+			boolean: true,
+			description: 'increase verbosity',
+			alias: 'v',
+		},
+		commandOptions.hostOSAccess,
+		{
+			signature: 'noproxy',
+			boolean: true,
+			description: `don't use the proxy configuration for this connection. \
+Only makes sense if you've configured proxy globally.`,
+		},
+	],
+	async action(params, options, done) {
+		const childProcess = await import('child_process');
+		const Bluebird = await import('bluebird');
+		const resin = (await import('resin-sdk')).fromSharedOptions();
```

Comment:
Should we do `const { resin } = await import('../sdk');` like in `lib/actions/settings.ts`?